doc/readme/EXAMPLES.md

```diff
@@ -35,10 +35,14 @@ Alternatively, you see [INTERFACES.md](INTERFACES.md) on how to run them manuall
 For details, see `CMakeLists.txt` in `examples/meta/` for details, `generate.py` and `translate.py` in `examples/meta/generator/`.
 
 ## Adding new examples
-It is extremely simple to add a new example: simply create another `*.sg` file.
-We are currently porting all existing Python examples in the deprecated folder `examples/undocumented/python_modular` to the new system -- a copy-paste [entrance task](https://github.com/shogun-toolbox/shogun/issues/3555).
-
-If you are porting a new example, great! Better yet, add the [data](https://github.com/shogun-toolbox/shogun-data) so that we can do integration tests automatically as described in [DEVELOPING.md](DEVELOPING.md#testing).
+It is extremely simple to add a new example: simply create another `*.sg.in` file in the `examples/meta/src/*/` directory.
+For example, if you are porting `kernel-linear.py` then you should create `examples/meta/src/**kernel/linear.sg.in**`. To create the corresponding `.sg` file (which will be located in the build directory) run
+ ```
+ cmake .. && cmake --build . --target meta_examples
+ ```
+Please ensure `BUILD_META_EXAMPLES=ON` is set. We are currently porting all existing Python examples in the deprecated folder `examples/undocumented/python_modular` to the shogun's meta language. For further details have a look at [#3555](https://github.com/shogun-toolbox/shogun/issues/3555).
+
+If you are porting a new example, great! Better yet, add the corresponding data. Check [DEVELOPMENT.md](DEVELOPING.md###Integration_testing_of_results) under "Integration testing of results" to learn how to add integration data after porting meta example.
```

Comment:
isn't "under "Integration testing of results" redundant, as you already provide the respective href?

Reply:
True! But when you click on the link we are directed a bit above "Integration test results" , so I added the line to show them where exactly to look. More like a safety measure

Reply:
I just read the file as a whole. Almost all the things you added here are written above (cmake command, the flags, where files are located). This bloats the file. Further, if people are unable to locate the relevant section in the referenced develop readme, then all hope is lost.
Therefore, I am quite reluctant of merging this.